### PR TITLE
Enhancement/shortcut menu

### PR DESCRIPTION
**Pull request by dvantetering**
This branch has a menu which shows all the shortcuts in our application.

### CODE REVIEW

**Comment by ric-jon** [comment]
Code looks fine!

File: src/main/java/application/factories/MenuFactory.java [hunk 1]
```diff
@@ -124,6 +126,35 @@ private GridPane buildHelpGridPane2() {
         return grid2;
     }
 
+    private void showShortCutMenu() {
```
Comment (Warnars) [inline src/main/java/application/factories/MenuFactory.java]: Please add JDoc

File: src/main/java/application/factories/MenuFactory.java [hunk 2]
```diff
@@ -77,7 +77,7 @@ public MenuBar createMenu(MenuBar bar) {
     }
 
     private Menu initHelpMenu() {
```
Comment (ghost) [inline src/main/java/application/factories/MenuFactory.java:145]: Please add JDoc

File: src/main/java/application/factories/MenuFactory.java (hunk 1 above)
Comment (dvantetering) [inline src/main/java/application/factories/MenuFactory.java]: JDoc already added in other branch!

File: src/main/java/application/factories/MenuFactory.java [hunk 3]
```diff
@@ -153,85 +261,112 @@ private GridPane buildHelpGridPane() {
...
         );
     }
 
+    /**
+     * Create the menu for choosing a most recent GFF file
+     * @return the mostRecentGFFMenu
+     */
     private Menu initMostRecentGFFMenu() {
-        return initMostRecentMenu(RecentMenuTypes.GFF, mainController.getMostRecentGFF());
+        mostRecentGFF = initMostRecentMenu(RecentMenuTypes.GFF, mainController.getMostRecentGFF());
```
Comment (arbreurkes) [inline src/main/java/application/factories/MenuFactory.java:333]: Redundant object creation.

@nubcaker the objects aren't redundant, we need to create and instantiate them in order to access them when switching scenes. This is needed to enable and disable them. Same goes for the other two

File: src/main/java/application/factories/MenuFactory.java [hunk 4]
```diff
@@ -153,85 +261,112 @@ private GridPane buildHelpGridPane() {
...
+        return mostRecentGFF;
     }
 
+    /**
+     * Create the menu for choosing a most recent MetaData file
+     * @return the mostRecentMetaDataMenu
+     */
     private Menu initMostRecentMetadataMenu() {
-        return initMostRecentMenu(RecentMenuTypes.META_DATA, mainController.getMostRecentMetadata());
+        mostRecentMeta = initMostRecentMenu(RecentMenuTypes.META_DATA, mainController.getMostRecentMetadata());
```
Comment (arbreurkes) [inline src/main/java/application/factories/MenuFactory.java:342]: Redundant object creation.

File: src/main/java/application/factories/MenuFactory.java [hunk 5]
```diff
@@ -153,85 +261,112 @@ private GridPane buildHelpGridPane() {
...
+        return mostRecentMeta;
     }
 
+    /**
+     * Create the menu for choosing a most recent GFA file
+     * @return the mostRecentGFAMenu
+     */
     private Menu initMostRecentGFAMenu() {
-        return initMostRecentMenu(RecentMenuTypes.GFA, mainController.getMostRecentGFA());
+        mostRecentGFA = initMostRecentMenu(RecentMenuTypes.GFA, mainController.getMostRecentGFA());
```
Comment (arbreurkes) [inline src/main/java/application/factories/MenuFactory.java:351]: Redundant object creation.

File: src/main/java/application/factories/MenuFactory.java [hunk 6]
```diff
@@ -153,85 +261,112 @@ private GridPane buildHelpGridPane() {
...
+        return mostRecentGFA;
     }
 
+    /**
+     * Create the menu for choosing a most recent NWK file
+     * @return the mostRecentNWKMenu
+     */
     private Menu initMostRecentNWKMenu() {
-        return initMostRecentMenu(RecentMenuTypes.NWK, mainController.getMostRecentNWK());
+        mostRecentNWK = initMostRecentMenu(RecentMenuTypes.NWK, mainController.getMostRecentNWK());
```
Comment (arbreurkes) [inline src/main/java/application/factories/MenuFactory.java:360]: Redundant object creation.

**Comment by arbreurkes** [comment]
@dvantetering after reading your response, this branch is ready to merge.

**Pull request merged**